resources/schema.json

```diff
@@ -307,15 +307,31 @@
                     {
                       "required": [
                         "variables"
-                      ]
+                      ],
```

Comment:
I think this could be simplified to use `anyOf` since any combination of `variables` and `target` is valid.

```json
{
  "anyOf": [
    {
      "required": [
        "source"
      ]
    },
    {
      "required": [
        "sourceExpression"
      ]
    }
  ]
}
```

Reply:
simplified